tests/testthat/test_utils.R

```diff
@@ -598,3 +598,16 @@ test_that("levelIndex works", {
     res <- levelIndex(str, which = "last")
     expect_equal(res, c(a = 4, b = 8, c = 7, d = 11))
 })
+
+test_that(".spectra_from_data works", {
+    spctra <- spectra(tmt_erwinia_on_disk)
```

Comment:
BTW we should be careful writing tests that depend on `MSnExp` and `OnDiskMSnExp`. We need a few to be sure that `MSnExperiment` yield identical results. But if we are going to replace (and deprecate/defunct) `(OnDisk)MSnExp` in the far, far away future we have to rewrite all the tests.

Reply:
Of course. These are intermediates.